bin/pylal_cbc_cohptf_trig_combiner

```diff
@@ -335,11 +335,11 @@ def main(trigFiles, ifoTag, userTag, segdir, outdir, grbTag=None, jobTag=None,
   if shortSlides or timeSlides:
     for trig in trigs['ALL_TIMES']:
       slideID = int(trig.time_slide_id)
+      in_buffer = []
       for ifo in ifos:
         offset = slideDictList[slideID][ifo]
-        if trig.get_end()+offset in segs['buffer']:
-          break
-        else:
```

Comment:
I think this was an indentation error. If lines 342 and 343 are shifted one indent left (so the `else:` and `for ifo in ifos:` are on the same line, it should work. 

However, I'm also happy with the proposed fix (it's slightly slower, but clarity is more important than speed here). So you have my approval to go with whatever fix you prefer.